### PR TITLE
Remove deprecated cortexOrgId in prometheus scaler

### PR DESCRIPTION
**Pull request by dttung2905**
_Provide a description of what has been changed_

### Checklist

- [x] I have verified that my change is according to the [deprecations & breaking changes policy](https://github.com/kedacore/governance/blob/main/DEPRECATIONS.md)
- [x] Tests have been added
- [x] Changelog has been updated and is aligned with our [changelog requirements](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#Changelog)
- [x] A PR is opened to update the documentation on ([repo](https://github.com/kedacore/keda-docs)) *(if applicable)*
- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))

Fixes https://github.com/kedacore/keda/issues/5538

### CODE REVIEW

**Comment by JorTurFer** [comment]
Could you update docs as well?

File: CHANGELOG.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ Here is an overview of all new **experimental** features:
 
 ### Deprecations
 
-You can find all deprecations in [this overview](https://github.com/kedacore/keda/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Abreaking-change) and [join the discussion here](https://github.com/kedacore/keda/discussions/categories/deprecations).
+- **Prometheus Scaler**: Remove deprecated field `cortexOrgID` from Prometheus scaler ([#5538](https://github.com/kedacore/keda/issues/5538))
```
[the remark this answers is not in the document]
Comment (dttung2905) [reply CHANGELOG.md]: Its done :rocket:

**Review by JorTurFer — APPROVED** [review]

**Pull request merged**